### ISSUE TITLE
Table Panel: Pagination Resets with browser refresh and data refresh

### ISSUE THREAD

**Issue by codeincarnate** [grafana/grafana#61259]
**What happened**:

When refreshing a dashboard either through browser refresh or refreshing data using the built in refresh controls the table panel pagination will be reset to the first page.

The video below demonstrates this issue:

https://user-images.githubusercontent.com/199847/211763809-537446e3-7576-4b03-aee2-5f7f3be56a3b.mov

**What you expected to happen**:

The table panel pagination should be persisted (assuming there is adequate data in a update) with data refreshes.

**How to reproduce it (as minimally and precisely as possible)**:

Create a table panel and enable pagination on the panel. Navigate past the first page and then refresh the data.

**Anything else we need to know?**:

This may have been working at one point (see https://github.com/grafana/grafana/issues/14780), however it appears this may have been targeted at the old table panel and it's unclear if this was implemented with the new panel. 

**Environment**:
- Grafana version: `main`
- Data source type & version: CSV Test Data Source
- OS Grafana is installed on: Mac OS 
- User OS & Browser: Mac OS/Chrome

**Comment by codeincarnate** [comment]
My initial thought is that we probably want to persist table pagination state in the URL so that way linked dashboards will show the desired page of a table. We may want to extend share controls and give the option to strip out pagination information as well.

**Comment by supilee** [comment]
Do we plan to have this implemented for G10? right now (9.5.0-pre) it's still there.

**Comment by zoltanbedi** [comment]
@mdvictor is this now fixed by your pr that disabled the page index reset?

**Comment by zoltanbedi** [comment]
I think it is not a good idea to store it in the url as a dashboard can have multiple table panels with different pages selected. Maybe store it in the panel cfg but I don't see many people finding this issue so it is a low prio one IMO.

**Comment by mdvictor** [comment]
yeah, I also think page saving on browser refresh isn't a big issue, if one at all, in the same way column filtering isn't an issue for resetting on browser refresh
